06-func.md

```diff
@@ -137,11 +136,9 @@ The computer does this for historical reasons:
 integer operations were much faster on early machines,
 and this behavior is actually useful in a lot of situations.
 It's still confusing,
-though,
-so Python 3 produces a floating-point answer when dividing integers if it needs to.
+therefore, Python 3 produces a floating-point answer when dividing integers if it needs to.
```

Comment:
I'm not sure what sort of differentiation was made between integers and floating-point in the rest of the tutorials, but this statement is not exactly correct. Python 3 _always_ produces a floating point answer from division, not only "if it needs to".
```python
$ python3 -c 'print(3/3)'
1.0
```